pkg/api/org_users.go

```diff
@@ -64,7 +64,12 @@ func GetOrgUsersForCurrentOrgLookup(c *models.ReqContext) Response {
 		return Error(403, "Permission denied", nil)
 	}
 
-	orgUsers, err := getOrgUsersHelper(c.OrgId, c.Query("query"), c.QueryInt("limit"))
+	limit := c.QueryInt("limit")
+	if limit == 0 || limit > 100 {
+		limit = 100
+	}
+
+	orgUsers, err := getOrgUsersHelper(c.OrgId, c.Query("query"), limit)
```

Comment:
This API is used from the users tab under configuration which has no pagination currently, so this low limit here will be a problem

Reply:
I'm not quite sure about that, when I see the api calls from users tab under configuration it is `/api/org/users` which is mapped to method `GetOrgUsersForCurrentOrg`, the other api call when adding users to a team is `/api/org/users/lookup` which is mapped to `GetOrgUsersForCurrentOrgLookup`, and this last endpoint is being called only from `UserPicker` component :thinking: